qiita_pet/support_files/doc/source/tutorials/getting-started.rst

```diff
@@ -1,328 +0,0 @@
-for random access to sequences associated with samples, as well as
-per-sample statistics. This format originated from the need to fetch
-sequences associated with individual samples, which required substantial
-overhead when working with ASCII formatted sequence files such as fasta
-and fastq. The structure provided by HDF5 enables Qiita to rapidly
-access the sequence data for any sample, and additionally, to
-efficiently subset (potentially randomly) the corresponding sequences.
-
-HDF5 can be thought of internally as a filesystem, where directories are
-called “groups” and files are called “datasets.” In the HDF5 demux
-format, a sample is a group and the sequence data are decomposed into
-multiple datasets. Specifically, the following datasets are directly
-part of the sample group:
-
--  sequence, which contains the actual sequence data stored as a vector
-   of string.
--  qual, which contains the quality scores per sequence per nucleotide,
-   stored as a matrix of integers. Sequences that do not have quality
-   scores associated (e.g., sourced from a Sanger file) will have zeros
-   for all positions.
-
-Barcode details can be found under the “barcode” group of the sample.
-Within there are three datasets:
-
--  original, which contains the original barcodes associated with the
-   sequences stored as a vector of string.
--  corrected, which contains the corrected barcodes (e.g., the result of
-   a corrected substitution error within the barcode) associated with
-   the sequences stored as a vector of string.
--  error, which contain the number of observed barcode errors per
-   sequence stored as a vector of integer.
-
-All datasets within a sample are in index-order. In other words, the
-sequence at index zero corresponds to the quality at row zero,
-corresponds to the barcode at index zero, etc.
-
-Last, the following summary statistics are tracked per-sample
-(accessible via the group attributes) and per-file (accessible via the
-file attributes):
-
--  n, the number of sequences stored as an integer.
--  max, the maximum sequence length stored as an integer.
--  min, the minimum sequence length stored as an integer.
--  mean, the mean sequence length stored as a floating point value.
--  std, the standard deviation of sequence length stored as a floating
-   point value.
--  median, the median sequence length stored as a floating point value.
--  hist, a 10-bin histogram of sequence lengths stored as a vector of
-   integer.
--  hist\_edge, the edges of each bin in the sequence length histogram
-   stored as a vector of integer.
-
-Once you are happy with these files and you are ready for publication,
-you can contact one of the Qiita admins to submit to EBI, this process normally
-takes a couple of days but can take more depending on availability and how busy
-is the submitting queue.
-
-Study status
```

Comment:
Sounds reasonable. However, I think Figure 2 of https://qiita.ucsd.edu/static/doc/html/qiita-philosophy/index.html covers it, what do you think?